server/application/app.py

```diff
@@ -13,4 +15,17 @@ def create_app(config_name):
     config_module = f"application.config.{config_name.capitalize()}Config"
     app.config.from_object(config_module)
     app.register_blueprint(work_shift.blueprint)
+
+    @app.errorhandler(AuthenticationError)
```

Comment:
@Sai9797 I really would like for you to follow the example from the Clean Architectures in Python book. This code is not following that example. That example is nice, because it gives us a very clear step by step instructions for how to build a maintainable web application in Python.